cpp/src/arrow/python/pyarrow_api.h

```diff
@@ -17,66 +17,71 @@
 
 // DO NOT EDIT THIS FILE. Update from pyarrow/lib_api.h after pyarrow build
 
-/* Generated by Cython 0.25.2 */
+/* Generated by Cython 0.29 */
 
 #ifndef __PYX_HAVE_API__pyarrow__lib
 #define __PYX_HAVE_API__pyarrow__lib
 #include "Python.h"
+#include "pyarrow_lib.h"
```

Comment:
I realize this is Cython-generated code, but why is including this file becoming necessary? @scoder